assets/chezmoi.io/docs/user-guide/tools/diff.md

```diff
@@ -18,8 +18,8 @@ specify:
 !!! hint
 
     If you generate your config file from a config file template, then you'll
-    need to escape the `{{` and `}}` in your config file template so that they
-    appear in your generated config file.
+    need to escape the `{{` and `}}` as `{{"{{"}}` and `{{"}}"}}`. That way 
```

Comment:
Two small things:

1. There is trailing whitespace after `way`, causing the build failure.
2. Remove the `the` in this fragment (both files): `` the `{{` ``. 

If I look at this with my documentation editor hat on, I would probably rephrase the whole thing to

```markdown
If you generate your config file from a template, then you need to escape `{{` and `}}`
as `{{ "{{" }}` and `{{ "}}" }}` in your template so that your generated config file
contains `{{` and `}}` characters.
```

It *may* also be worth pointing people to the [delimiters template directive](https://www.chezmoi.io/reference/templates/directives/#delimiters) (`[delimiters template directives](reference/templates/directives/#delimiters)`)  as this can make the question of `{{` and `}}` irrelevant.

This comment applies to both files you have modified.

Reply:
Reapplying this comment as the whitespace has been removed.